README.md

```diff
@@ -1 +1,23 @@
 # hello_project
+> This is my first repository for the hello_microverse project.
+
+## Built With
+
+- HTML and CSS
+
+## Getting Started
+
+**This is an example of how I have to set up my projects.
+
+
+## Authors
+
+👤 **Andres Condezo**
+
+- GitHub: [@andres-condezo](https://github.com/andres-condezo)
+- Twitter: [@andres_condezo](https://twitter.com/andres_condezo)
+- LinkedIn: [andres-condezo](https://linkedin.com/in/andres-condezo)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It seems you forgot to add the MIT file! Please fix the broken link :link: . You can find the file in this [template](https://github.com/microverseinc/readme-template#readme)

Reply:
Right!, now I upload it.